app/services/albums.js

```diff
@@ -1,6 +1,6 @@
 const request = require('request-promise');
-const errors = require('errors.js');
-const logger = require('/logger/index.js');
+const errors = require('../errors');
+const logger = require('../logger/index.js');
```

Comment:
Remember require only logger, like `require('../logger')`, index is inferred automatically when you're requiring a folder